mlflow/environment_variables.py

```diff
@@ -841,6 +841,36 @@ def get(self):
     "MLFLOW_LOCK_MODEL_DEPENDENCIES", False
 )
 
+#: Specifies whether to enable delta archival for trace data to TraceServer.
+#: When set to True, traces will be exported to both MLflow V3 and TraceServer in parallel.
+#: (default: ``False``)
+MLFLOW_TRACING_ENABLE_DELTA_ARCHIVAL = _BooleanEnvironmentVariable("MLFLOW_TRACING_ENABLE_DELTA_ARCHIVAL", False)
+
+#: Specifies the name of the table to ingest spans into for TraceServer.
+#: This is only used when MLFLOW_TRACING_ENABLE_DELTA_ARCHIVAL is set to True.
+#: (default: ``None``)
+MLFLOW_TRACING_DELTA_ARCHIVAL_SPANS_TABLE = _EnvironmentVariable("MLFLOW_TRACING_DELTA_ARCHIVAL_SPANS_TABLE", str, None)
+
+#: Specifies the personal access token for authentication for TraceServer.
+#: This is only used when MLFLOW_TRACING_ENABLE_DELTA_ARCHIVAL is set to True.
+#: (default: ``None``)
+MLFLOW_TRACING_DELTA_ARCHIVAL_TOKEN = _EnvironmentVariable("MLFLOW_TRACING_DELTA_ARCHIVAL_TOKEN", str, None)
```

Comment:
Is this necessary, or can we reuse the token that MLflow already retrieves for REST API requests in `mlflow.utils.databricks_utils.get_databricks_host_creds()`?